Models/Entities/PSPDatabaseContext.cs

```diff
@@ -8,10 +8,18 @@ public class PSPDatabaseContext : DbContext
     public PSPDatabaseContext(DbContextOptions<PSPDatabaseContext> options)
     : base(options)
     {
-        ChangeTracker.QueryTrackingBehavior = QueryTrackingBehavior.NoTracking;
```

Comment:
Sorry, took over your PR :D. @Azuolas3 is there a reason this was added, since it broke some of my logic with orders. Can it be removed?